pages/browse/index.js

```diff
@@ -1,18 +1,20 @@
 import { useEffect, useState } from 'react'
 import { useRouter } from 'next/router'
-import {  Error, Item, SearchBar } from '@scientist-softserv/webstore-component-library'
+import { useSession } from 'next-auth/react'
+import { Error, Item, SearchBar } from '@scientist-softserv/webstore-component-library'
 import { configureErrors, configureServices, useFilteredWares } from '../../utils'
 
 const Browse = () => {
   const router = useRouter()
+  const { data: session } = useSession()
   const [query, setQuery] = useState('')
   const existingQuery = router.query.q
 
   useEffect(() => {
     if (existingQuery) setQuery(existingQuery)
   }, [existingQuery])
 
-  const { wares, isLoading, isError } = useFilteredWares(query)
+  const { wares, isLoading, isError } = useFilteredWares(query, session?.accessToken)
```

Comment:
`useSession` can only be used inside a component, so we need to pass the user's access token into every one of our functions now. previously we've been using a hard coded access token.